### PR TITLE
[offscale-libetcd-cpp] no absolute paths

### PR DESCRIPTION
**Pull request by autoantwort**
For #20469

### CODE REVIEW

**Comment by autoantwort** [comment]
It seems that `vcpkg_cmake_configure` does not change the cwd in contrast to `vcpkg_configure_cmake` on windows. 
The line `file(MAKE_DIRECTORY generated)` generates the following error on windows but not on other platforms:
```
CMake Error at CMakeLists.txt:13 (file):
  file attempted to create a directory:
  C:/Users/LeanderSchulten/git_projects/vcpkg/buildtrees/offscale-libetcd-cpp/src/fdb87cf638-60a9ea69fd.clean/generated
  into a source directory.
```

File: ports/offscale-libetcd-cpp/portfile.cmake [hunk 1]
```diff
@@ -4,15 +4,16 @@ vcpkg_from_github(
...
+    PATCHES
+        fix-dependency-grpc.patch
+        install-debug.patch
 )
 
-vcpkg_configure_cmake(
-    SOURCE_PATH ${SOURCE_PATH}
-    PREFER_NINJA
+vcpkg_cmake_configure(
+    SOURCE_PATH "${SOURCE_PATH}"
```
Comment (JackBoosY) [inline ports/offscale-libetcd-cpp/portfile.cmake:13]:
```suggestion
    SOURCE_PATH "${SOURCE_PATH}"
    DISABLE_PARALLEL_CONFIGURE
```

**Review by JackBoosY — CHANGES_REQUESTED** [review]

**Review by JackBoosY — APPROVED** [review]

**Comment by autoantwort** [comment]
@PhoebeHui  I think you can remove the tag `requires:author-response`

**Review by JackBoosY — APPROVED** [review]

**Review by PhoebeHui — APPROVED** [review]

**Comment by PhoebeHui** [comment]
@autoantwort, sorry for late response, and thanks Jack for helping review PR!

**Pull request merged**